### DISCUSSION TITLE
dxvk-1.9 released

### DISCUSSION THREAD

**Discussion by thinkprocess** [Sporif/dxvk-async#15] [General]
Hello, people! Are there any plans to make a patch for the new version?

Sorry. You've already done it)))

**Comment by Sporif** [comment]
Yes I added a new release 5 minutes ago. No changes were needed, seems to be still compatible from my quick testing.

**Reply by thinkprocess** [reply]
Thank you very much! I will also test it, and if I find any problems, I will definitely report them in detail.

**Comment by Ryusennin** [comment]
Hi Sporif,

May I ask a new build based on the latest master version?
https://github.com/doitsujin/dxvk/actions/runs/984728257

It fixes serious regression problems in some games using the Unreal engine.
Thanks in advance!

**Reply by Sporif** [reply]
Sure, here it is: [dxvk-async-1.9_39_g2ff8b42f.tar.gz](https://github.com/Sporif/dxvk-async/files/6739818/dxvk-async-1.9_39_g2ff8b42f.tar.gz)

**Reply by ghost** [reply]
@Sporif 
Hi ☺️ I was in the same boat. The latest commit to dxvk master fixes an issue for me with some games. Would you kindly patch the latest commit with Async.

Thank you Sporif 🙏🙏

**Reply by Sporif** [reply]
Here you go: [dxvk-async-1.9_r56_g5f1629f2.tar.gz](https://github.com/Sporif/dxvk-async/files/6810636/dxvk-async-1.9_r56_g5f1629f2.tar.gz).
